packages/cli/src/commands/serve.ts

```diff
@@ -2,13 +2,12 @@ import Fastify from 'fastify';
 import path from 'path';
 import ora from 'ora';
 import childProcess from 'child_process';
-import { constants } from '@pipcook/pipcook-core';
+import { PIPCOOK_LOGS } from '@pipcook/pipcook-utils';
```

Comment:
Constants should be in core?

Reply:
These constants are path related which requires path module. So I move them into utils

Reply:
As I have thought at https://github.com/alibaba/pipcook/pull/195#discussion_r429023922, if the core is used internally, it depends on if we need to provide these paths to plugin developers.